newIDE/app/src/AssetStore/CreditsPackages/CreditsPackageStoreContext.js

```diff
@@ -4,21 +4,43 @@ import {
   listListedCreditsPackages,
   type CreditsPackageListingData,
 } from '../../Utils/GDevelopServices/Shop';
+import CreditsPackagesDialog from '../../Credits/CreditsPackagesDialog';
+import CreditsUsageDialog from '../../Credits/CreditsUsageDialog';
+
+type CreditsUsageDialogOptions = {|
+  title: React.Node,
+  message: React.Node,
+  onConfirm: () => Promise<void>,
+  successMessage: React.Node,
+|};
 
 type CreditsPackageStoreState = {|
   fetchCreditsPackages: () => void,
   creditsPackageListingDatas: ?Array<CreditsPackageListingData>,
   error: ?Error,
+  openCreditsPackageDialog: (
+    options: ?{ missingCredits?: number, showCalloutTip?: boolean }
```

Comment:
```suggestion
    options: ?{| missingCredits?: number, showCalloutTip?: boolean |}
```

Reply:
Given that you use it in other places, I think you should create a `CreditsPackageDialogOpeningOptions` type